modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -26,9 +26,9 @@ In the table, features are marked with the following statuses:
 |===
 |{gitops-shortname} |Argo CD CLI |Helm |Kustomize |Argo CD |Argo Rollouts |Dex |RH SSO |Argo CD Agent |{ocp}
 
+|1.19.0 |3.1.9  TP |3.18.4 GA |5.7.0 GA |3.1.9 GA |1.8.3 GA |2.43.0 GA |NA |0.5.1 TP |4.14, 4.16-4.20
```

Comment:
```suggestion
|1.19.0 |3.1.9  TP |3.18.4 GA |5.7.0 GA |3.1.9 GA |1.8.3 GA |2.43.0 GA |NA |0.5.1 GA |4.14, 4.16-4.20
```

Since the argocd agent is going GA?